cmd/subctl/root.go

```diff
@@ -42,3 +52,41 @@ func Execute() {
 		os.Exit(1)
 	}
 }
+
+func detectGlobalnet() {
+	status := cli.NewReporter()
+	status.Start("Trying to detect Globalnet settings")
+
+	clientProducer, err := client.NewProducerFromRestConfig(framework.RestConfigs[framework.ClusterA])
+	exit.OnError(status.Error(err, "Error creating client producer"))
+
+	operatorClient := clientProducer.ForOperator()
+
+	submariner, err := operatorClient.SubmarinerV1alpha1().Submariners(constants.OperatorNamespace).Get(
+		context.TODO(), constants.SubmarinerName, v1.GetOptions{})
+	if k8serrors.IsNotFound(err) {
+		exit.OnError(status.Error(err, "The Submariner resource was not found. Either submariner has not"+
```

Comment:
We don't really need to output the K8s error here.

```suggestion
		exit.WithMessage("The Submariner resource was not found. Either submariner has not"+
```